release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -390,6 +390,57 @@ Linux Control Group version 2 (cgroup v2) is now generally available in {product
 
 [id="ocp-4-13-monitoring"]
 === Monitoring
+The monitoring stack for this release includes the following new and modified features.
+
+[id="ocp-4-13-monitoring-updates-to-monitoring-stack-components-and-dependencies"]
+==== Updates to monitoring stack components and dependencies
+This release includes the following version updates for monitoring stack components and dependencies:
+
+* Alertmanager to 0.25.0
+* kube-state-metrics to 2.8.1
+* node-exporter to 1.5.0
+* prom-label-proxy to 0.6.0
+* Prometheus to 2.42.0
+* prometheus-adapter to 0.10.0
```

Comment:
no need to mention prometheus-adapter, since the 4.12 version is also 0.10.0
https://docs.openshift.com/container-platform/4.12/release_notes/ocp-4-12-release-notes.html#ocp-4-12-monitoring-updates-to-monitoring-stack-compnents-and-dependencies

Reply:
Removed